app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class EventDecorator < Draper::Decorator
+  delegate_all
+
+   def self.collection_decorator_class
+    PaginatingDecorator
+  end
+
+  def beginning_at
+    object.beginning_at.strftime("%Y-%m-%d %R")
```

Comment:
please create a datetime format in locales and use http://api.rubyonrails.org/classes/ActionView/Helpers/TranslationHelper.html#method-i-localize helper here